src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -204,6 +207,13 @@ class ReferralService(
       }
     }
 
+    update.serviceCategoryIds?.let {
+      val serviceCategories = serviceCategoryRepository.findByIdIn(it)
+      if (!referral.intervention.dynamicFrameworkContract.contractType.serviceCategories.containsAll(serviceCategories)) {
+        errors.add(FieldError(field = "serviceCategories", error = Code.INVALID_SERVICE_CATEGORY_FOR_CONTRACT))
```

Comment:
i think the field name is incorrect in this error. it should be `serviceCategoryIds`?